Practice2/README.md

```diff
@@ -34,9 +34,18 @@
     * images/
   * styles/
     * base/
+      * all.css
       * base.css
       * reset.css
     * components/
+      * common/
+        * all.css
+        * button.css
+        * heading-default.css
+        * heading-medium.css
+        * heading.css
+        * input-stay-in-touch.css
```

Comment:
This is not a suitable file for the common folder. If you put any item in the common, it should be not specific to stuff like `stay-in-touch`.

Reply:
Fixed